Automattic/newspack-blocks
# HPB: support for The Events Calendar events

### All Submissions:






### Changes proposed in this Pull Request:



Adds support for The Events Calendar in Homepage Posts Block.

### How to test the changes in this Pull Request:

1. Install [The Events Calendar](https://wordpress.org/plugins/the-events-calendar/) and add some events 
2. Insert a HP block on a page and choose "Events" 
3. Observe that where the post publish date is rendered for regular posts, event date and venue name (if set) are rendered for TEC's Events
4. Publish and observe the same same effect on the front-end

### Other information:
